recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma-salvator-x-h3.cfg

```diff
@@ -98,8 +98,8 @@ vcpus = 4
 # docs/misc/xl-network-configuration.markdown
 vif = [ 'backend=DomD,bridge=xenbr0,mac=08:00:27:ff:cb:ce' ]
 
-vdispl = [ 'backend=DomD,be-alloc=0,connectors=1000:1024x768' ]
+vdispl = [ 'backend=DomD,be-alloc=0,connectors=1000:1920x1080;1001:1088x1487' ]
```

Comment:
Are you sure about an odd height of a secondary screen?

Reply:
Yes, this is size of application AGL layout.